src/style.css

```
@@ -1 +1,59 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+ 
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  font-size: 12px;
+  font-weight: 500;
+  margin-bottom: 50px;
+}
+
+.nav__item:not(:last-child) {
+  padding-right: 20px;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+  padding: 0;
+}
+
+.nav__link {
+  text-decoration: none;
+  line-height: 0;
+  display: inline-block;
+  text-transform: uppercase;
+  color: #060b35;
+  padding: 23px 0;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active::after {
+  display: block;
+  content: "";
+  background-color: #00acdc;
+  height: 4px;
+  border-radius: 8px;
+  margin-top: 19px;
+}
```

```suggestion
.is-active::after {
  position: absolute;
  display: block;
  content: "";
  background-color: #00acdc;
  height: 4px;
  border-radius: 8px;
  bottom: 0;
}
```
and position relative for `nav__link`